subsys/bluetooth/shell/bt.c

```diff
@@ -1518,7 +1527,10 @@ static int cmd_per_adv_sync_create(const struct shell *shell, size_t argc,
 
 	create_params.options = options;
 
-	bt_le_per_adv_sync_cb_register(&per_adv_sync_cb);
+	if (!cb_registered) {
+		bt_le_per_adv_sync_cb_register(&per_adv_sync_cb);
+	}
+	cb_registered = true;
```

Comment:
You did not delete this after adding in bt_ready.

Reply:
Good catch :)